contrib/babelfishpg_common/src/collation.c

```diff
@@ -1587,3 +1587,15 @@ babelfish_update_server_collation_name(PG_FUNCTION_ARGS)
 	MemoryContextSwitchTo(oldContext);
 	PG_RETURN_VOID();
 }
+
+Oid like_cid = InvalidOid;
```

Comment:
I'm fine with the current approach, but usually hooks should be order-independent and call prev_hooks inside. An alternative could be to define `tsql_like_cid` in `like_match.c` and use it there.